### DISCUSSION TITLE
Upload file using temporary url generated by PresignedPutObject

### DISCUSSION THREAD

**Discussion by ggdream** [minio/minio#14147] [Q&A] [answered]
Hi minio team, sorry to ask the question here, but I can't really find the answer. The question is please give an example of implementing the header requirement with `curl`.
I found a lot of client examples and official web examples, but none of them work in my flutter app. I can't speculate on the upload format, so I can't know who the problem is.

**Comment by vadmeste** [comment, marked as answer]
@ggdream it is only a plain HTTP PUT request where the URL is the presigned URL and the body contains the file that you want to upload. No header is needed since all information are in the presigned URL itself.

**Comment by kannappanr** [comment]
Are you looking for something like this

curl -X PUT "https://play.min.io/src/issue?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=minioadmin%2F20220121%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20220121T064050Z&X-Amz-Expires=1000&X-Amz-SignedHeaders=host&X-Amz-Signature=3968511e5954c2edec0861eecf2c4e3c85af3edbaef6d779dab1ba462f3a7d77" -d "test data" -H "Content-Type: application/json"
